core/app/models/spree/order.rb

```diff
@@ -374,7 +374,7 @@ def finalize!
       # update payment and shipment(s) states, and save
       updater.update_payment_state
       shipments.each do |shipment|
-        shipment.update!(self)
+        shipment.ready
```

Comment:
will need to remove this as it moves shipments to next state before we hear back from signifyd